### DISCUSSION TITLE
Generate pages and text lines Preview

### DISCUSSION THREAD

**Discussion by LeonvanHeerden** [DrewThomasson/ebook2audiobook#349] [Ideas]
Option to generate a preview of X pages or X number of lines. 
Rather than waiting for the entire book generation to find out there is an issue.

**Comment by ROBERT-MCDOWELL** [comment]
It's a useful feature indeed.

**Comment by ROBERT-MCDOWELL** [comment]
it's on rails, date of release not known.

**Comment by ROBERT-MCDOWELL** [comment]
will be available very soon.

**Comment by ROBERT-MCDOWELL** [comment]
fully implemented in version 26.4.12 via checkbox "chapters preview" in GUI gradio mode.
